src/Symfony/Component/HttpFoundation/Tests/StreamedJsonResponseTest.php

```diff
@@ -37,24 +37,50 @@ public function testResponseObjectsList()
                 '_embedded' => [
                     'articles' => $this->generatorArray('Article'),
                 ],
+                'placeholder' => '__symfony_json__',
```

Comment:
This should be in an own test case and the `placeholder` should be in front of the \Generator e.g.:

```php
    public function testResponsePlaceholderExists()
    {
        $content = $this->createSendResponse(
            [
                'placeholder' => '__symfony_json__',
                '_embedded' => [
                    'articles' => $this->generatorArray('Article'),
                ],
             ],
        );
        
        $this->assertSame('{"placeholder":"__symfony_json__", "_embedded":{"articles":[{"title":"Article 1"},{"title":"Article 2"},{"title":"Article 3"}]}}', $content);
    }
}
```

This is the case which will fail and why I introduced `uniqid` instead of a hardcoded placeholder to have a none  non guessable placeholder.

Reply:
It does not fail, because they are detected [here](https://github.com/alexander-schranz/symfony/pull/1/files#diff-88676ff0fb58128ade940542c46b371799cd3d2cb30f4b1d83d5e5beb165a661R80) then reinjected [here](https://github.com/alexander-schranz/symfony/pull/1/files#diff-88676ff0fb58128ade940542c46b371799cd3d2cb30f4b1d83d5e5beb165a661R94-R95)

Reply:
I created a dedicated test case.